### ISSUE TITLE
[BUG] in some environment MythicCrucible could not be init when oraxen marked softdepend on it

### ISSUE THREAD

**Issue by killerprojecte** [oraxen/oraxen#1610]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I am using ProtocolLib 5.0+
- [X] I have checked if any similar bug reports exist

### Description

If you start with the default oraxen, MythicCrucible will be initialized before MythicMobs. If I remove MythicCrucible from oraxen's softdepend, the order will remain normal and MythicCrucible will load normally.

This problem cannot be reproduced on other platforms and I don't know what causes this problem.
As long as MythicCrucible does not appear in oraxen's softdepend, everything will work fine.

### Steps to reproduce

Running oraxen with MythicCrucible 2.1.0 and MythicMobs 5.7.2

### System Report

[00:50:51 INFO]: ### System Report
[00:50:51 INFO]: **Plugin Versions:**
[00:50:51 INFO]: - Oraxen: 1.186.1
[00:50:51 INFO]: - ProtocolLib: 5.2.0-SNAPSHOT-679
[00:50:51 INFO]:
[00:50:51 INFO]: **Server Information:**
[00:50:51 INFO]: - Software: Purpur
[00:50:51 INFO]: - Version: git-Purpur-2062 (MC: 1.20.1)
[00:50:51 INFO]:
[00:50:51 INFO]: **System Information:**
[00:50:51 INFO]: - OS: Windows Server 2022
[00:50:51 INFO]: - OS Version: 10.0
[00:50:51 INFO]: - Architecture: amd64

### How breaking is the bug?

Minor Bug - Plugin completely functional, but features have non-working aspects

**Comment by Th0rgal** [comment]
Did you notice any bug removing MythicCrucible from softdepends?

**Comment by killerprojecte** [comment]
> Did you notice any bug removing MythicCrucible from softdepends?

No, I don't use MythicCrucible features in oraxen, I only use it in MythicMobs for item skills.

**Comment by Th0rgal** [comment]
This appears to be a platform-specific plugin loading order issue. The reporter confirmed that removing MythicCrucible from softdepend resolved it, and no actual Oraxen functionality was affected. Plugin load ordering is managed by the server platform, not Oraxen. Closing as a platform-specific edge case.
